guides/doc-Provisioning_Guide/topics/Networking.adoc

```diff
@@ -381,6 +381,85 @@ Before you begin, ensure that the following conditions are met:
 * The NIC is capable of PXE booting. For more information, see http://ipxe.org/appnote/hardware_drivers
 * The NIC is compatible with iPXE
 
+==== Example WIP of Lzap's Blogpost
+
+Using {Project} you can set up and use iPXE if you have the following restrictions that prevent you from using PXE:
+
+* On networks with unmanaged DHCP servers,
+* If the PXE service is blacklisted on your network or restricted by a firewall
+* When the TFTP UDP-based protocol is not reliable, for example, on low-bandwidth networks
+
+With {Project}, you can set up PXE over TFTP to download the `ipxe.efi` file. When the file downloads, all communication continues using HTTP. {Project} uses the iPXE provisioning script either to load an operating system installer or the next entry in the boot order.
+
+When an EFI host boots from PXE, the `ipxe.efi` file changes the process to iPXE. BIOS systems use the `undionly.kpxe` file.
+
+You can use this workflow with or without discovery phase. If you use discovery, use the HTTPBoot  {SmartProxy} module to host the {Project} Discovery image to keep the {Project} discovery image close to the managed nodes.
+
+.Security Information
+
+The iPXE binary in {RHEL} is built without security features, for this reason, you can use HTTP, and cannot use HTTPs. All security-related features of iPXE in {RHEL} are not supported. For more information, see https://access.redhat.com/solutions/3483601[Red Hat Enterprise Linux HTTPS support in iPXE].
+
+.Prerequisite
+
+Ensure that the hardware that you use is compatible with iPXE.
+
+.Procedure
+
+. Copy the `ipxe.efi` file to the TFTP directory. For BIOS systems, copy the `undionly.kpxe` file.
++
+----
+# cp /usr/share/ipxe/undionly.kpxe /var/lib/tftpboot/undionly.0
+# cp /usr/share/ipxe/ipxe.efi /var/lib/tftpboot/
+----
++
+. Because the feature cannot be enabled using the foreman-installer command, you must edit the `settings.d/httpboot.yml` configuration file to enable:
```

Comment:
For Satellite 6.7 installer will definitely work, I've tested this on Friday. Not sure for 6.6, but I'd probably document installer and maybe a note that for older Satellite versions 6.5-6.6 it must be enabled manually.

Reply:
I will remove this note and only publish the changes to these docs for the 6.7 version onwards.